### ISSUE TITLE
Baguette Model is too suggestive about what is "the middle"

### ISSUE THREAD

**Issue by bastianilso** [med-material/La-baguette#11]
The baguette looks really nice, but the curvyness on the top of it, suggests very easily to you where the center, which might be problematic given the purpose of the baguette test (assessing peoples ability to cut in the center).

![image](https://user-images.githubusercontent.com/3967945/56740134-9f697f00-6770-11e9-9115-a9e04fc5526a.png)

**Comment by EkatiSag** [comment]
That´s true. Maybe to avoid that we could make a really simple 3d model with no curves and just play with the brushes to make it realistic, but still I don´t know if it will be that good

**Comment by bastianilso** [comment]
Hendrik suggests using diagonal cuts as they are much better at concealing the true mid-point.
![image](https://user-images.githubusercontent.com/3967945/56783451-1a23b000-67ec-11e9-8444-6c7b460a5aef.png)
